.github/workflows/ok-to-test.yml

```diff
@@ -23,7 +27,7 @@ jobs:
         private_key: ${{ secrets.PRIVATE_KEY }}
 
     - name: Slash Command Dispatch
-      uses: peter-evans/slash-command-dispatch@v1
+      uses: peter-evans/slash-command-dispatch@v3
```

Comment:
Does this version require `.args.named.sha` instead of `.sha` [here](https://github.com/imjohnbo/ok-to-test/pull/22/files?diff=unified&w=0#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bR29)?

Reply:
I'm not quite sure which line you are refering to?

If you mean `github.event.client_payload.pull_request.head.sha`, then this is fine as is. As can be seen from this file of examples: https://github.com/peter-evans/slash-command-dispatch/blob/7dceb85f0d8928300b4278657b2e389e6b25766b/docs/examples.md

Reply:
Along with bumping `peter-evans/slash-command-dispatch@v1` to `peter-evans/slash-command-dispatch@v3`, I see you're [proposing](https://github.com/imjohnbo/ok-to-test/pull/22/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bR29) this change; just want to double check this is necessary because of the version bump?

```diff
- github.event.client_payload.slash_command.sha != ''
+ github.event.client_payload.slash_command.args.named.sha != ''
```